style.css

```diff
@@ -13,151 +43,147 @@ html {
 
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
   background-color: var(--background);
   color: var(--text);
-  font-family: 'Roboto', sans-serif;
+  letter-spacing: 0.06rem;
 }
 
 a {
   color: var(--accent);
   text-decoration: none;
 }
 
-#header {
+p {
+  line-height: 1.5;
+}
+
+header {
   position: fixed;
-  box-sizing: border-box;
-  z-index: 10;
+  top: 0;
+  left: 0;
+  right: 0;
   background-color: var(--background);
-  padding: 0 5vw;
-  padding-top: 5vh;
-  padding-bottom: 2vh;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  padding: 1.5rem 2.5rem 1.5rem 2rem;
 }
 
-#brand {
-  height: 2rem;
-  content: url('assets/brand.svg');
-  margin-left: 0.5em;
+nav {
+  display: flex;
+  column-gap: 2.5rem;
 }
 
-#navbar a {
+nav a {
   color: var(--text);
   text-transform: uppercase;
-  text-decoration: none;
-  font-size: 1.8rem;
-  margin: 0 1rem;
+  font-size: 1.5rem;
 }
 
-.container {
+section {
   min-height: 100vh;
-  width: 100vw;
-  max-width: 100%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
 }
-/* startpage */
-#get #download {
+
+/* Start Page */
+.download-links {
   display: flex;
+  column-gap: 3rem;
+  flex-wrap: wrap;
   justify-content: center;
 }
 
-#get h1 {
+.download h1 {
   text-transform: uppercase;
-  margin-bottom: 5vh;
+  margin-bottom: 2rem;
   font-size: 3rem;
 }
 
-#get #download a img {
-  width: 15em;
-  margin: 0 0.5em;
-}
-/* about secion */
-#about {
-  justify-content: space-between;
+.download-links img {
+  width: 15rem;
 }
 
-#slogan {
-  margin-top: 42vh;
+/* About Section */
+#about {
+  padding-top: 1rem;
```

Comment:
The `Privacy Simplified` container goes a bit to bottom with `padding-top: 4rem;`